### PR TITLE
Simplified Caching

### PR DESCRIPTION
**Pull request by whalesalad**
I had a difficult time expressing some of my thoughts on this, so I figured a PR might be easier to interpret.

### CODE REVIEW

File: flutil/db.py [hunk 1]
```diff
@@ -12,8 +12,9 @@
 class EnvironmentVariableNotFoundException(Exception):
     pass
 
+
 class PoolManager:
-    connections = {}
+    _connections = {}
 
     @staticmethod
     def get_url_from_environment(name=None):
```
Comment (dbjohnson) [inline flutil/db.py]: nit: suggest moving this to the bottom of the class since it's a helper method

File: flutil/db.py [hunk 2]
```diff
@@ -43,22 +44,31 @@ def from_name(cls, name=None, cached=True):
         if not name:
             name = 'DATABASE_URL'
 
-        url, env = cls.get_url_from_environment(name)
+        url, env = PoolManager.get_url_from_environment(name)
```
Comment (dbjohnson) [inline flutil/db.py]: nit: you have cls available here

**Comment by dbjohnson** [comment]
+1

**Comment by hfukada** [comment]
+1

**Pull request merged**